release-3.7.0-announcement.md

```diff
@@ -0,0 +1,31 @@
+We're excited to announce the availability of Jamulus 3.7.0!
+
+This release re-organizes Server lists to make room for more servers:
+- Default has been renamed to Any Genre 1.
+- Any Genre has been renamed to Any Genre 2.
+- Any Genre 3 has been added.
+- Choral/Barbershop has been added.
+- Classical/Folk/Choral has been renamed to Classical/Folk.
+
+Besides that, numerous usability improvements, enhancements, bugfixes and optimizations have been integrated.
+Please find all the details in the [Changelog](https://github.com/jamulussoftware/jamulus/releases/tag/r3_7_0).
+
+Downloads (primarily on Github, alternatively on [SourceForge](https://sourceforge.net/projects/llcon/files/latest/download)):
+- [Windows](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_win.exe)
+- [Mac](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_mac.dmg)
+- [Debian/Ubuntu (amd64)](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_ubuntu_amd64.deb), alternative: [headless version](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_headless_3.7.0_ubuntu_amd64.deb)
+- [Source code](https://github.com/jamulussoftware/jamulus/archive/r3_7_0.tar.gz)
+
+This is the first release after the founder, developer and project lead of Jamulus, Volker Fischer (@corrados), [decided to leave the project](https://github.com/jamulussoftware/jamulus/discussions/928).
+We want to use this opportunity to thank Volker again. Creating Jamulus and pushing it to the point where it is today was and is a huge accomplishment! Thanks, Volker!
+
+Starting with this release, the project will be lead by the Main Developers Team.
```

Comment:
Minor point: in the irregular mess that is English, the past tense and past participle of "lead" is "led".

Reply:
Thanks, fixed. BTW, I'm happy about such pointers. :)